hyperspy/io.py

```diff
@@ -112,6 +112,23 @@ def load(filenames=None,
         For SEMPER unf- and EMD (Berkley)-files, if True (default is False)
         additional information read during loading is printed for a quick
         overview.
+        
+    downsample : int (1–4095)
+        For Bruker bcf files, if set to integer (>=2) (default 1)
```

Comment:
The language might need some improvement. Would any native speaker like to help (e.g. @tjof2).

If not, I can help making it more spanglish...